experiments/long_runs/land.jl

```diff
@@ -0,0 +1,660 @@
+    )
+
+    energy_args = (parameters = Canopy.BigLeafEnergyParameters{FT}(ac_canopy),)
+
+    # Canopy component args
+    canopy_component_args = (;
+        autotrophic_respiration = autotrophic_respiration_args,
+        radiative_transfer = radiative_transfer_args,
+        photosynthesis = photosynthesis_args,
+        conductance = conductance_args,
+        hydraulics = plant_hydraulics_args,
+        energy = energy_args,
+    )
+
+    # Other info needed
+    shared_params = Canopy.SharedCanopyParameters{FT, typeof(earth_param_set)}(
+        z0_m,
+        z0_b,
+        earth_param_set,
+    )
+
+    canopy_model_args = (;
+        parameters = shared_params,
+        domain = ClimaLand.obtain_surface_domain(domain),
+    )
+
+    # Integrated plant hydraulics and soil model
+    land_input = (atmos = atmos, radiation = radiation, runoff = runoff_model)
+    land = SoilCanopyModel{FT}(;
+        soilco2_type = soilco2_type,
+        soilco2_args = soilco2_args,
+        land_args = land_input,
+        soil_model_type = soil_model_type,
+        soil_args = soil_args,
+        canopy_component_types = canopy_component_types,
+        canopy_component_args = canopy_component_args,
+        canopy_model_args = canopy_model_args,
+    )
+
+    Y, p, cds = initialize(land)
+
+    init_soil(ν, θ_r) = θ_r + (ν - θ_r) / 2
+    Y.soil.ϑ_l .= init_soil.(ν, θ_r)
+    Y.soil.θ_i .= FT(0.0)
+    T = FT(276.85)
+    ρc_s =
+        Soil.volumetric_heat_capacity.(
+            Y.soil.ϑ_l,
+            Y.soil.θ_i,
+            soil_params.ρc_ds,
+            soil_params.earth_param_set,
+        )
+    Y.soil.ρe_int .=
+        Soil.volumetric_internal_energy.(
+            Y.soil.θ_i,
+            ρc_s,
+            T,
+            soil_params.earth_param_set,
+        )
+    Y.soilco2.C .= FT(0.000412) # set to atmospheric co2, mol co2 per mol air
+    Y.canopy.hydraulics.ϑ_l.:1 .= plant_ν
+    evaluate!(Y.canopy.energy.T, atmos.T, t0)
+
+    set_initial_cache! = make_set_initial_cache(land)
+    exp_tendency! = make_exp_tendency(land)
+    imp_tendency! = ClimaLand.make_imp_tendency(land)
+    jacobian! = ClimaLand.make_jacobian(land)
+    set_initial_cache!(p, Y, t0)
+
+    # set up jacobian info
+    jac_kwargs =
+        (; jac_prototype = ImplicitEquationJacobian(Y), Wfact = jacobian!)
+
+    prob = SciMLBase.ODEProblem(
+        CTS.ClimaODEFunction(
+            T_exp! = exp_tendency!,
+            T_imp! = SciMLBase.ODEFunction(imp_tendency!; jac_kwargs...),
+            dss! = ClimaLand.dss!,
+        ),
+        Y,
+        (t0, tf),
+        p,
+    )
+
+    updateat = Array(t0:(3600 * 3):tf)
+    updatefunc = ClimaLand.make_update_drivers(atmos, radiation)
+
+    # ClimaDiagnostics
+
+    space = domain.space.subsurface
+
+    nc_writer = ClimaDiagnostics.Writers.NetCDFWriter(space, outdir)
+
+    diags =
+        ClimaLand.CLD.default_diagnostics(model, t0; output_writer = nc_writer)
```

Comment:
model = land

Reply:
diags =
        ClimaLand.CLD.default_diagnostics(model, t0; output_writer = nc_writer, output_vars = "short")

see the changes in src/diagnostics/default_diagnostics.jl in PR #682 (for now, copy the PR files to this PR?)